lib/pycriu/criu.py

```diff
@@ -45,7 +46,14 @@ def __init__(self, sk_path):
 
     def connect(self, daemon):
         self.sk = socket.socket(socket.AF_UNIX, socket.SOCK_SEQPACKET)
-        self.sk.connect(self.comm)
+        try:
+            self.sk.connect(self.comm)
+            
+        except FileNotFoundError:
+            raise FileNotFoundError("Socket file not found.")
+        
```

Comment:
Maybe we can cleanup excess spaces on this line and line 51.

In `12d6e3f96 ("pycriu: better socket error handling")`.